src/Button/DefaultButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Button from './Button';
+import '../index';
+
+interface DefaultButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: string
```

Comment:
Limiting the children to just 'string' is fine for right now but we might need to revisit this later on if we decide to allow icons to be parsed down to the buttons as well.

Reply:
@xylocopaCaerulea so they type should be any?